docs/reference.md

```diff
@@ -106,7 +106,7 @@ are relevant for _all_ build job configurations.
 * `job`: the job to execute from the build configuration. If not specified, the job will be `build`.
 * `--config`: the config file to read. Optional, and if does not exist, will search
   for it, as described [above](#configuration-file).
-* `packagesToBuild`: a set of package directories or package names. If a directory, it MUST
+* `packagesToBuild`: a set of package directories or shortcuts or the entire package names. If a directory, it MUST
```

Comment:
A bit laconic as information. More information would be appreciated. It doesn't have to be in this paragraph. It could be in a separate paragraph underneath it).

Reply:
Updated:)